Build/Common.proj

```diff
@@ -73,7 +72,7 @@
   </PropertyGroup>
 
   <!-- References -->
-  <Import Project="$(DlrBuildDir)\$(ReferencedPlatform).props" />
```

Comment:
I'm not super familiar with how it works, but msbuild does an "outer" build and an "inner" build. In the "outer" build pass TargetFramework is not set. You can try to read about it if you want, I don't understand much of it: https://oren.codes/2017/01/04/multi-targeting-the-world-a-single-project-to-rule-them-all/#how-it-works